ts/db.go

```diff
@@ -133,14 +133,19 @@ func (db *DB) StoreData(r Resolution, data []proto.TimeSeriesData) error {
 	// InternalMergeRequest, probably because it cannot be part of a
 	// transaction. Look into batching this.
 	for _, kv := range kvs {
-		if err := db.kv.Run(client.Call{
+		// Note, this looks like a batch, but isn't a batch because we only add a
+		// single request to it.
+		b := &client.Batch{}
```

Comment:
Not easily because `proto.BatchRequest` cannot contain a `proto.InternalMergeRequest`. We could change it to do so, but that's more involved and outside of the scope of this change (hence the TODO).